browser.css

```diff
@@ -56,6 +56,16 @@ img.larrybird {
 	padding-left: 80px !important;
 }
 
+/* scale down the profile picture image in expanded mode and add a margin of 10px */
```

Comment:
I don't think the comment should reference the exact `margin`. If we ever change the `margin` and forget the comment it'll be confusing.